packages/cli/src/commands/validate.ts

```diff
@@ -0,0 +1,157 @@
+import { command } from "../utils/command.js";
+import { C, print, S } from "../utils/screens.js";
+import { LabelsValidator, MonokleValidator } from "validation";
+import { failure, success } from "./validate.io.js";
+import { readFile } from "fs/promises";
+import { extractK8sResources, File } from "../utils/extract.js";
+
+type Options = {
+  path: string;
+};
+
+export const validate = command<Options>({
+  command: "validate [path]",
+  describe: "Validate your Kubernetes resources",
+  builder(args) {
+    return args.positional("path", { type: "string", demandOption: true });
```

Comment:
Piping stdin will allow us to test more scenarios easier. `cat $path | monokle validate` and `helm template | monokle validate`.